PfR_loading_data.R

```diff
@@ -230,6 +230,18 @@ quiz_done_data <- plhdata_org %>%
   pivot_longer(cols = -app_user_id, names_to = "name", values_to = "response") %>%
   group_by(name, response) %>%
   summarise(`Count` = n(), .groups = "drop") %>%
-  pivot_wider(names_from = response, values_from = `Count`, values_fill = list(`Count` = 0)) %>%
-  mutate(name = tools::toTitleCase(gsub("_", " ", name)))
-
+  mutate(name = stringr::str_remove(name, "rp-contact-field.quiz_question_"))
+  #pivot_wider(names_from = response, values_from = `Count`, values_fill = list(`Count` = 0)) %>%
+  #mutate(name = tools::toTitleCase(gsub("_", " ", name)))
+
+library(readxl)
+Quiz_responses <- read_excel("C:/Users/Cabrine/Downloads/PfR in app data collected via App.xlsx", 
```

Comment:
Move the `PfR in app data collected via App.xlsx` data into the same file as your `excel_shiny.xlsx` data
Then call from the same location